packages/desktop-client/src/components/schedules/SchedulesTable.tsx

```
@@ -196,6 +170,127 @@ export function ScheduleAmountCell({
+      <Field width={110} name="date">
+        {schedule.next_date
+          ? monthUtilFormat(schedule.next_date, dateFormat)
+          : null}
+      </Field>
+      <Field width={120} name="status" style={{ alignItems: 'flex-start' }}>
+        <StatusBadge status={statuses.get(schedule.id)} />
+      </Field>
+      <ScheduleAmountCell amount={schedule._amount} op={schedule._amountOp} />
+      {!minimal && (
+        <Field width={80} style={{ textAlign: 'center' }}>
+          {schedule._date && schedule._date.frequency && (
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)